provider/src/main/java/com/raygun/raygun4android/RaygunClient.java

```diff
@@ -55,6 +56,43 @@ public static void init(Application application, String apiKey) {
         init(application, apiKey, null);
     }
 
+    /**
+     * Initializes the Raygun client with applicationContext and your Raygun API key. The version
+     * transmitted will be the value of the versionName attribute in your manifest element. This
+     * function should be used by 3rd party library.
+     *
+     * @param Context The Android applicationContext
+     * @param apiKey An API key that belongs to a Raygun application created in your dashboard
+     */
+    public static void init(Context applicationContext, String apiKey) {
+        TimberRaygunLoggerImplementation.init();
+
+        RaygunLogger.d("Configuring Raygun4Android (v" + RaygunSettings.RAYGUN_CLIENT_VERSION + ")");
+
+        RaygunClient.applicationContext = applicationContext;
+
+        if (apiKey == null || apiKey.trim().isEmpty()) {
+            RaygunClient.apiKey = readApiKey(getApplicationContext());
+        } else {
+            RaygunClient.apiKey = apiKey;
+        }
+
+        RaygunClient.appContextIdentifier = UUID.randomUUID().toString();
+
+        if (version == null || version.trim().isEmpty()) {
```

Comment:
The new constructor for applicationContext doesn't pass in version, so this code will be problematic and not work.